LinkedList/Node.cs

```diff
@@ -6,6 +6,8 @@
     public class Node
     {
         // Class Node hold element from the Double Linked Listed
+        //Test comment :)
+        //Final comment 
```

Comment:
comments have to begin with space // comment

Reply:
emi dobre